src/services/incident.services.js

```diff
@@ -52,8 +52,24 @@ const incidentServices = {
         }
         redFlagComment.comment = comment;
         return redFlagComment || {};
-    }
+    },
 
+    // @describe delete a red-flag record
+    // @public api/v1/red-flags/:id
+    removeARedFlag(id) {
+        const flagId = incidents.find(incident => incident.id === +id);
+        if (!flagId) {
+            return new customError(401, 'This Record does not exist');
+        }
+        const index = incidents.indexOf(flagId);
+        console.log(index);
+        if (index !== -1) { 
```

Comment:
Since you have the check on line 61 `if (!flagId) {` it is safe to assume that line 66 will always be true. So remove the if statement